MovieGallery/Pages/Search.cshtml.cs

```diff
@@ -20,7 +20,9 @@ public class SearchModel : PageModel
         public Shows ShowsResult = new Shows();
         public Shows ShowsJson { get; set; }
 
-        public void OnGet(string SearchbyYear)
+
```

Comment:
Created OnGet method to load data when the page is initially loaded and no search term is passed.